main.py

```
@@ -0,0 +1,146 @@
+    email: EmailStr
+    nickname: str = Field(min_length=4, max_length=8)
+    profile_image: Optional[str] = None
+    created_at: datetime
+
+@app.post("/users/", response_model=UserResponse)
+async def signup(user_in: CreateUser):
+    return{
+        "email": user_in.email,
+        "nickname": user_in.nickname,
+        "profile_image": user_in.profile_image,
+        "created_at": datetime.now()
```

Ruff (0.14.11)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)